packages/transactions/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -1,131 +1,57 @@
-import React, { FC, useEffect, useState } from 'react';
+import React, { FC } from 'react';
```

Comment:
an extra thing in this file - can you make the first column (name) sortable?
currently has `sortable` as false, from when it was the id column.
there's also something up when sortable is false - the column has the sort hover, so it looks like you can sort it, but nothing happens.